src/main/flight/mixer.c

```diff
@@ -833,7 +829,7 @@ void mixTable(void)
             if (isFailsafeActive) {
                 mixConstrainMotorForFailsafeCondition(i);
             } else if (feature(FEATURE_3D)) {
-                if (throttle >= flight3DConfig->deadband3d_high)  {
+                if (throttlePrevious >= (rxConfig->midrc + flight3DConfig->deadband3d_throttle)) {
```

Comment:
You are using opposite test than on https://github.com/cleanflight/cleanflight/pull/1792/files#diff-95e70d63cf1e155a276b2ad2a8e7a947R795.
It is not problem now (throttlePrevious is always outside deadband3d_throttle), but can cause problems if code is changed